config-model/src/main/java/com/yahoo/vespa/model/container/Container.java

```diff
@@ -239,8 +240,7 @@ public boolean requiresConsecutivePorts() {
      */
     public int getPortCount() {
         int httpPorts = (getHttp() != null) ? 0 : numHttpServerPorts + 2; // TODO remove +2, only here to keep irrelevant unit tests from failing.
-        int rpcPorts = (rpcServerEnabled()) ? numRpcServerPorts : 0;
-        return httpPorts + rpcPorts;
+        return httpPorts + numMessageBusPorts() + numRpcPorts();
```

Comment:
Can the port count be extracted from `portsMeta`? This method and the getters below seems to duplicate the logic/info already stored in `portsMeta`.

Reply:
No, because getPortCount is called before the `portsMeta` is populated.